docs/imageproxy.rst

```diff
@@ -0,0 +1,140 @@
+===========
+Image Proxy
+===========
+
+.. contents::
+
+The image proxy is an independent service to manipulate images in various ways
+(e.g. crop or resize images).
+
+The commands and the image URL have to be provided as query parameters and the
+whole query string has to be signed to disallow others to penetrate the service.
+
+
+Base URLs
+=========
+
+Each environment has it's own image proxy.
+
+On DEV environment::
+
+    https://imageproxy-iris-dev.lovelysystems.com
+
+On STAGING environment::
+
+    https://imageproxy-iris-stg.lovelysystems.com
+
+On PRODUCTION environment::
+
+    https://images.petition.ch
+
+
+Query Parameters
+================
+
+The general parameters are described in the `Pilbox Documentation
+<https://github.com/agschwender/pilbox#general-parameters>`_.
+
+Additional information:
+
+- `url`: The url of the image to be resized. The service is configured to use
+  the S3 base URL by default. For images uploaded by the :doc:`files` you
+  can skip the hostname and only use the file's ID with a leading slash.
+  It's also possible to use images from other locations by providing the
+  full URL.
+- `client`: The client parameter is required and must contain the value
```

Comment:
"must contain the value" indicates, that ``fooirisbar`` would also work. does it?
if not, we should be more exactly here, e.g. "must have the value ``iris``.